bot.py

```diff
@@ -1272,7 +1272,7 @@ def check(m):
     if not session[0] or message.author.id not in session[1] or session[1][message.author.id][1] not in ['shaman', 'crazed shaman'] or not session[1][message.author.id][0]:
         return
     if session[2]:
-        await reply(message, "You may only give totems during the night.")
+        await reply(message, "")
```

Comment:
I think you forgot to fill this in? Also you might want to commit the lang files too